plugins/connections/ui/src/main/java/org/pentaho/di/vfs/connections/ui/dialog/ConnectionDelegate.java

```diff
@@ -27,9 +27,13 @@
 import org.pentaho.di.connections.ConnectionManager;
 import org.pentaho.di.connections.ui.dialog.ConnectionDeleteDialog;
 import org.pentaho.di.connections.ui.tree.ConnectionFolderProvider;
+import org.pentaho.di.core.bowl.Bowl;
```

Comment:
Copyright date is not updated

Reply:
good catch. Added to https://github.com/pentaho/pentaho-kettle/pull/9319